app/src/main/res/raw/ccind.cer

```diff
@@ -0,0 +1,30 @@
+-----BEGIN CERTIFICATE-----

```

Comment:
@akashkj Is this a certificate specific to Dimagi ? Does this expire ever ?

Reply:
This certificate seem to be of custom domain where an instance of HQ is hosted currently. @RawatH Why is this certificate added, can you please tell?

Reply:
Removed